diff --git a/lxd/instance/drivers/driver_qemu.go b/lxd/instance/drivers/driver_qemu.go
index 40516bfec..5b572c815 100644
--- a/lxd/instance/drivers/driver_qemu.go
+++ b/lxd/instance/drivers/driver_qemu.go
@@ -34,7 +34,7 @@ import (
 	"golang.org/x/sys/unix"
 	"gopkg.in/yaml.v2"
 
-	"github.com/lxc/lxd/client"
+	lxd "github.com/lxc/lxd/client"
 	agentAPI "github.com/lxc/lxd/lxd-agent/api"
 	"github.com/lxc/lxd/lxd/apparmor"
 	"github.com/lxc/lxd/lxd/cgroup"
@@ -1533,11 +1533,11 @@ func (d *qemu) Start(stateful bool) error {
 	// This ensures that if the guest initiates a reboot that the SHUTDOWN event is generated instead with the
 	// reason set to "guest-reset" so that the event handler returned from getMonitorEventHandler() can restart
 	// the guest instead.
-	err = monitor.SetAction(map[string]string{"reboot": "shutdown"})
-	if err != nil {
-		op.Done(err)
-		return fmt.Errorf("Failed setting reboot action: %w", err)
-	}
+	// err = monitor.SetAction(map[string]string{"reboot": "shutdown"})
+	// if err != nil {
+	// 	op.Done(err)
+	// 	return fmt.Errorf("Failed setting reboot action: %w", err)
+	// }
 
 	_ = op.Reset() // Reset timeout to default.
 
